apps/settings/js/panels/achievements/panel.js

```diff
@@ -0,0 +1,19 @@
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+  var AchievementsList = require('panels/achievements/achievements_list');
+  var SettingsListener = require('shared/settings_listener');
+
+  return function ctor_achievements_panel() {
+    var achievementsList;
+
+    return SettingsPanel({
+      onInit: function(panel) {
+        achievementsList = AchievementsList(panel);
+        SettingsListener.observe('achievements', achievementsList.achievements,
```

Comment:
Following the comment above, using `achievementsList.achievements` as a default value sets an `ObservableArray` to `achievementsList.achievements` while it is expected to be an object.